packages/manager-components/src/components/templates/update-name-modal/update-name-modal.component.tsx

```diff
@@ -112,7 +145,7 @@ export const UpdateNameModal: React.FC<UpdateNameModalProps> = ({
         {cancelButtonLabel || t('updateModalCancelButton')}
       </OsdsButton>
       <OsdsButton
-        disabled={isLoading || undefined}
+        disabled={isLoading || (pattern && isPatternError) || undefined}
```

Comment:
```suggestion
        disabled={isLoading || isPatternError || undefined}
```

If there is no `pattern` defined, `isPatternError` will be always `false` so no need to test pattern in this case

Reply:
👍